File: LP_C++/lista_loop/loop_1.py
DEN = float(1)
DIV = float(1)
X = float (0)

while DEN<=99 and DIV<=50:
    X=DEN/DIV
    print("%.0f/%.0f= %.2f"%(DEN,DIV,X))
    DIV += 1
    DEN += 2

File: LP_C++/lista_loop/loop_1.cpp
#include <stdio.h>
#include <stdlib.h>


int main(int argc, char const *argv[])
{
    float den,div,x;
    den = 1;
    div = 1;
    x = 0;

    while (den<=99 && div<=50)
    {
        x=den/div;
        printf("%.0f/%.0f= %f\n",den,div,x);
        div++;
        den+=2;
    }
    

    return 0;
}
